content/integrations/awsrds.md

```diff
@@ -91,62 +95,58 @@ There are 3 options for monitoring RDS instances. You can choose to use standard
   1.  Ensure RDS is checked in the AWS Integration tile.
 
 * **Native Database Integration**
```

Comment:
Let's make these H3s or H4s instead of bullet points. Same with the Configuration section.

Then, in Configuration section, the body of each H3 should not use bullet points or numbering, unless there are many steps (i.e. 15 steps), e.g. instead of:

```
* **Standard RDS Integration**
 		 
   If you haven't already, set up the [Amazon Web Services integration first](/integrations/aws).
```

```

Let's have:

```
#### Standard RDS Integration

If you haven't already, set up the [Amazon Web Services integration first](/integrations/aws).
```

Overall, this document uses bullet points way too much.